packages/react-native/ReactAndroid/src/main/jni/react/turbomodule/ReactCommon/TurboModuleManager.cpp

```diff
@@ -130,8 +130,9 @@ void TurboModuleManager::registerNatives() {
 }
 
 TurboModuleProviderFunctionType TurboModuleManager::createTurboModuleProvider(
-    bool enableSyncVoidMethods) {
+    jsi::Runtime *runtime, bool enableSyncVoidMethods) {
   return [turboModuleCache_ = std::weak_ptr<ModuleCache>(turboModuleCache_),
+          runtime,
```

Comment:
the `runtime` could be a dangling pointer in theory. but the lambda would run on js thread and there's the check. so it should not hit to handling pointer crash.

```java
    if (!turboModuleCache || !jsCallInvoker || !nativeMethodCallInvoker ||
        !delegate || !javaPart) {
      return nullptr;
    }
```